keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -52,46 +74,51 @@ systems to help with key rotation and protect against key exfiltration.
 ### Goals
 
 1. Credential rotation without client restart.
-1. Support standard key management solutions.
-1. Support standard token-based protocols.
-1. Provisioning logic lives outside of Kubernetes codebase.
-1. Kubernetes interface is vendor-neutral.
+2. Support standard key management solutions.
+3. Support standard token-based protocols.
+4. Provisioning logic lives outside of Kubernetes codebase.
+5. Kubernetes interface is vendor-neutral.
+6. Deprecation of `gcp` and `azure` authentication options (`keystone` has
+   already been deprecated and removed).
 
 ### Non-Goals
 
 1. Exfiltration protection built into Kubernetes.
-1. Kubernetes triggering rotation.
-1. Deprecation of existing authentication options.
+2. Kubernetes triggering rotation.
+3. Deprecation of the `oidc` authentication option.
 
 ## Proposal
 
 A new authentication flow in libraries around `kubeconfig` based on
 executables. Before performing a request, client executes a binary and uses its
 output for authentication.
 
-There are 2 modes of authentication:
+There are two modes of authentication:
 
-1. bearer tokens
-1. mTLS
+1. Bearer tokens
+2. mTLS
 
-Provider response is cached and reused in future requests.
+Provider response is cached in-memory by the client and reused in future
+requests (no caching is done across different executions of the client).
```

Comment:
Updated wording around caching to indicate in-memory only.

Reply:
The plugin should still be able to cache on disk (or some protected hardware store).
Maybe clarify that the plugin can internally do whatever.

Reply:
Done.